host/xtest/pkcs11_1000.c

```diff
@@ -8016,3 +8016,442 @@ static void xtest_pkcs11_test_1025(ADBG_Case_t *c)
 
 ADBG_CASE_DEFINE(pkcs11, 1025, xtest_pkcs11_test_1025,
 		 "PKCS11: EDDSA key generation and signing");
+
+#define RSA_AES_MAX_KEY_SIZE 32
+
+#define RSA_AES_WRAP_KEY(vect) \
+	{ \
+		vect ## _modulus, ARRAY_SIZE(vect ## _modulus), \
+		vect ## _pub_exp, ARRAY_SIZE(vect ## _pub_exp), \
+		vect ## _priv_exp, ARRAY_SIZE(vect ## _priv_exp), \
+		vect ## _prime1, ARRAY_SIZE(vect ## _prime1), \
+		vect ## _prime2, ARRAY_SIZE(vect ## _prime2), \
+	}
```

Comment:
Prefer with explicit field labels:
```c
#define RSA_AES_WRAP_KEY_VECT(vect) { \
		.modulus = vect ## _modulus,			\
		.modulus_len = ARRAY_SIZE(vect ## _modulus),	\
		.pub_ext = vect ## _pub_exp,			\
		.pub_ext_len = ARRAY_SIZE(vect ## _pub_exp),	\
		.priv_ext = vect ## _priv_exp,			\
		.priv_ext_len = ARRAY_SIZE(vect ## _priv_exp),	\
		.prime1 = vect ## _prime1,			\
		.prime1_len = ARRAY_SIZE(vect ## _prime1),	\
		.prime2 = vect ## _prime2,			\
		.prime1_len = ARRAY_SIZE(vect ## _prime2),	\
	}

Reply:
done in [a4f3f71](https://github.com/OP-TEE/optee_test/pull/627/commits/a4f3f7118946a5f2be78e74c1bc82c65445f9752)